netlogo-gui/src/test/plot/PlotManagerTests.scala

```diff
@@ -44,6 +44,36 @@ class PlotManagerTests extends SimplePlotTest {
     assertResult(None)(manager.currentPlot)
   }
 
+    test("More Current Plot") {
```

Comment:
Spacing is off here.

Reply:
I've fixed the whitespace issue and rebased the commit messages.  I tried adding more context and a reason for the commit.